doc/users.rst

```diff
@@ -1,602 +1,303 @@
-       bostony_df, test_size=0.2)
-
-
-Then initialize a default Foreshadow object with a sklearn estimator such as XGBoost. We want this
-process to be fast so we can iterate, so for the time being we will override the default TPOT model selection,
-ensembling and hyperparameter optimization for regression problems with a simple default XGBoost regressor.
+   shadow = Foreshadow(problem_type=ProblemType.CLASSIFICATION, estimator=LogisticRegression())
 
 .. code-block:: python
 
-    shadow = fs.Foreshadow(estimator=XGBRegressor())
-
-Then fit the train data on that object
+   _ = shadow.fit(X_train, y_train)
 
+Making predictions
+~~~~~~~~~~~~~~~~~~
 .. code-block:: python
 
-    shadow.fit(X_train, Y_train)
-
-You now have an initial pipeline. Lets see how it did and serialize it to a JSON file so we can look at it.
+   predictions = shadow.predict(X_test)
 
 .. code-block:: python
 
-    # Score the pipeline
-    shadow.score(X_test, y_test)
-
-    # Serialize the pipeline
-    x_proc = shadow.X_preparer.serialize()
-    y_proc = shadow.y_preparer.serialize()
-
-    # Write the serialized pipelines to file
-    json.dump(x_proc, open("x_proc.json", "w"), indent=2)
-    json.dump(y_proc, open("y_proc.json", "w"), indent=2)
+   predictions.head()
 
-Now we have two pipeline configurations, one for our X data and one for our Y data. We also have an initial idea
-of how well the initial pipeline performed.
+Use the trained estimator to compute the evaluation score.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Let's suppose that you want to experiment with a different scaler. Open the configuration JSON and make this change. (Look to the `Configuration`_ section for more details on this)
-
-For example, add the following snippet to the bottom or x_proc.json
-
-.. code-block:: python
-
-    "combinations": [
-        {
-          "columns.CHAS.0": "['NumericIntent', 'CategoricalIntent']"
-        }
-     ]
-
-Now let's re-create the Foreshadow object with your changes.
+Note that the scoring method is defined by the selected estimator.
 
 .. code-block:: python
 
-    import json
+   shadow.score(X_test, y_test)
 
-    # Load in the configs from file
-    x_proc = json.load(open("x_proc.json", "r"))
-    y_proc = json.load(open("y_proc.json", "r"))
 
-    # Create the preprocessors
-    x_processor = fs.Preprocessor(from_json=x_proc)
-    y_processor = fs.Preprocessor(from_json=y_proc)
+You can inspect and change Foreshadow's decision
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    # Create the foreshadow object
-    shadow = fs.Foreshadow(X_preparer=x_processor, y_preparer=y_processor, estimator=XGBRegressor())
+Foreshadow uses a machine learning model to power the auto intent resolving step. As a user, you may not agree with the decision made by Foreshadow. The following APIs allow you to inspect the decisions and change them if you have a different opinion.
```

Comment:
We might want to add a section to briefly introduce the major functionality of Foreshadow at the beginning and point that one of the features is automatic intent resolving (we might need to explain what we mean by intent as well). With that foundation established, it will be easier to just say the decisions can be changed by users here.